source/Halibut.SampleServer/Program.cs

```diff
@@ -13,7 +13,7 @@ public static void Main(string[] args)
             Console.Title = "Halibut Server";
             var certificate = new X509Certificate2("HalibutServer.pfx");
 
-            var endPoint = new IPEndPoint(IPAddress.Any, 8433);
+            var endPoint = new IPEndPoint(IPAddress.IPv6Any, 8433);
```

Comment:
And apparently even accessing `DualMode` if the listener is only ipv4 throws an exception.